src/features/content-explorer/content-explorer/ContentExplorerActionButtons.js

```diff
@@ -73,10 +75,20 @@ const ContentExplorerActionButtons = ({
     };
 
     const renderStatus = () => {
-        const chosenItems = getChosenItemsFromSelectedItems(selectedItems);
-        const statusMessage = (
-            <FormattedMessage {...messages.numSelected} values={{ numSelected: chosenItems.length }} />
-        );
+        const numChosenItems = getChosenItemsFromSelectedItems(selectedItems).length;
```

Comment:
nit: if you rename this to `numSelected`, you don't need to transform it the values object